docs/user/overview.md

```diff
@@ -0,0 +1,61 @@
+
+In [supported environments](#supported-environments), the installer is also capable of provisioning the underlying infrastructure for the cluster. It is recommended that most users make use of this functionality in order to avoid having to provision their own infrastructure themselves. In unsupported environments or scenarios in which the created infrastructure is incompatible, the installer can stop short of creating the infrastructure, allowing the user to provision their own infrastructure using the cluster assets generated by the installer.
+
+
+## Cluster Installation Process ##
+
+OpenShift 4 is unique in that its management extends all the way down to the operating system itself. In order to make this possible, every machine boots with a configuration which references resources hosted in the cluster its joining. This allows the cluster to control how new machines should look and allows the cluster to vary that over time as updates are applied. A downside to this approach, however, is that new clusters have no way of starting without external help - every machine in the as-of-yet-created cluster is waiting on the as-of-yet-created cluster.
+
+OpenShift 4 breaks this dependency loop using a sacrificial bootstrap machine. This bootstrap machine is booted with a concrete [Ignition Config][ignition] which describes how to create the cluster. This machine acts as the remote arm of the installer, created in the target environment for the sole purpose of bringing up the rest of the cluster.
+
+The main asset generated by the installer is the Ignition Configs for the bootstrap, master, and worker machines. Given these three configs (and correctly configured infrastructure), it is possible to start an OpenShift cluster. The process for bootstrapping a cluster roughly looks as follows:
+
+  1. The bootstrap machine boots and starts hosting the remote resources required for the master machines to boot.
+  2. The master machines fetch the remote resources from the bootstrap machine and finish booting.
+  3. The master machines use the bootstrap node to form an etcd cluster.
+  4. The bootstrap node starts a temporary Kubernetes control plane using the newly-created etcd cluster.
+  5. The temporary control plane schedules the long-running control plane to the master machines.
+  6. The temporary control plane shuts down, yielding to the long-running control plane.
+  7. The bootstrap node injects OpenShift-specific components into the newly formed control plane.
+  8. The installer then tears down the bootstrap node.
+
+The result of this bootstrapping process is a fully running OpenShift control plane. The cluster will then download and configure remaining components needed for the day-to-day operation, including the creation of worker machines in supported environments.
```

Comment:
"control plane" -> "cluster"?

Reply:
the result of bootstrapping is a control plane, CVO continues to roll out the cluster even after the bootstrap node is gone.